### DISCUSSION TITLE
Provide Table Preview from the Window > New menu

### DISCUSSION THREAD

**Discussion by MikeCarlo** [TabularEditor/TabularEditor3#1048] [Ideas]
While I like the Pivot Grid, I am unable to see the information within a Table unless I add a measure to calculate.  

I would prefer to have a new view that would grab all the data from the table. Similar to how Power BI Desktop has the table view.  I would really like it to operate the same way.  Where I could select the Arrow in Desktop and filter the table based on selected options.  

If a full table can't be provided, then I'd like the Pivot Grid to show me a list of values found in a column.

**Comment by otykier** [comment]
Have you tried right-clicking on a table and choosing “Preview Data”? :-)

**Comment by MikeCarlo** [comment]
I did not notice this feature.  This is exactly what I would like but I feel like I would have found this in the Window -> New -> Table Preview option…

Then potentially drag over a table into the window to render a table.

Other side note, you can filter options inside the table the icon’s are a bit scrunched together.

Without Hover:
***@***.***D75D15.4DC03EE0]

When you hover the column the filter options show up:
***@***.***D75D15.4DC03EE0]

Hard to discover.

It’s also a bit challenging to select the sort option vs. the Filter column option when they are so small / close together.  Would recommend persisting the Filter Icon on every column.  Then adding the sort Icon when the column is sorted.

Lastly, Can I Auto Width Multiple Columns. I would like that ability as well.

Mike

**Comment by otykier** [comment]
Instead of providing an option for dragging a table into the Table Preview, I can display a dialog asking you to choose which table you want to preview. How does that sound?

Also, could you provide a screenshot of the icons that are scrunched together? Not sure what you're referring to.

**Comment by MikeCarlo** [comment]
The icons I was referring to were are shown in the two screen shots.

[Image.jpeg]
It’s hard to click on the filter icon.
One it only shows when you hover a column.  And I kept trying to click the filter icon but instead I kept sorting the table column.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by otykier** [comment]
Could you add the image to the thread on GitHub https://github.com/TabularEditor/TabularEditor3/issues/68 directly? For some reason, I can't view them when they're attached to an e-mail reply.

**Comment by MikeCarlo** [comment]
oh, odd, sorry, I will add them here. 

![image](https://user-images.githubusercontent.com/25160592/121540598-3b47ba80-c9cc-11eb-8888-490d8e78ce81.png)
